src/core/mutatus.scala

```diff
@@ -157,6 +152,26 @@ case class Service(readWrite: Datastore) {
   def read: DatastoreReader = readWrite
 }
 
+object Service {
+
+  /** provides a default instance of the GCP Datastore service */
+  lazy val default: Service = Service(
+    DatastoreOptions.getDefaultInstance.getService
+  )
+
+  /** Service instance which does not connect to any GCP Datastore, designed to be used in tests */
+  lazy val noop = Service {
```

Comment:
If you move this one to a different object (or nest it inside another object) and make it implicit, you could get the "no-op" behaviour just with,
```
import Service.noop._
```